modules/nf-core/simpleaf/index/tests/main.nf.test

```diff
@@ -67,20 +67,14 @@ nextflow_process {
         then {
             assertAll(
                 { assert process.success },
-                { assert snapshot(process.out.versions).match() },
                 { assert file("${process.out.index.get(0).get(1)}/piscem_idx_cfish.json").exists() },
                 { assert file("${process.out.index.get(0).get(1)}/piscem_idx.ctab").exists() },
                 { assert file("${process.out.index.get(0).get(1)}/piscem_idx.ectab").exists() },
                 { assert file("${process.out.index.get(0).get(1)}/piscem_idx.json").exists() },
                 { assert file("${process.out.index.get(0).get(1)}/piscem_idx.refinfo").exists() },
                 { assert file("${process.out.index.get(0).get(1)}/piscem_idx.sshash").exists() },
-                { assert file("${process.out.index.get(0).get(1)}/simpleaf_index.json").exists() }
-                // { assert snapshot(
-                //     path("${process.out.index.get(0).get(1)}/piscem_idx.ctab"),
-                //     path("${process.out.index.get(0).get(1)}/piscem_idx.json"),
-                //     path("${process.out.index.get(0).get(1)}/piscem_idx_cfish.json"),
-                //     process.out.versions)
-                //     .match() }
+                { assert file("${process.out.index.get(0).get(1)}/simpleaf_index.json").exists() },
```

Comment:
maybe a stupid question, but all of these files are unstable, i.e. you can't just snapshot them as a whole?